src/blockfrost/lib/Convex/Blockfrost/Types.hs

```diff
@@ -208,15 +207,27 @@ toAssetId = \case
 toAddress :: (C.IsCardanoEra era) => Address -> Maybe (C.AddressInEra era)
 toAddress (Address text) = C.deserialiseAddress (C.proxyToAsType Proxy) text
 
+serialiseToBech32 :: (C.SerialiseAsRawBytes a) => Text.Text -> a -> Text.Text
```

Comment:
Doesn't this function exist in `cardano-api`?

Reply:
yes but it is not exported anymore in the most recent version :(

Reply:
Add a TODO to remove this once it is added again? In the latest `main`, it *is* exported.